bylaws.html

```diff
@@ -11,99 +11,99 @@ <h1>CouchDB Bylaws</h1>
 
-<p>Your role is bestowed on you by your peers in recognition of your past contributions to the project and your position of trust within the community. It is not tied to your job, your current employer, or your current activity level. We are interested in you as an individual and we understand that your interactions with the project may change over time.
+<p>Your role is bestowed on you by your peers in recognition of your past contributions to the project and your position of trust within the community. It is not tied to your job, your current employer, or your current activity level. We are interested in you as an individual, and we understand that your interactions with the project may change over time.
 
-<p>Roles are never rescinded because of inactivity, unless that inactivity is causing a problem for the project. Fortunately, our decision making process means that inactivity is very rarely a problem. Roles will be rescinded if the <a href="#pmc">Project Management Committee</a> (or <a href="#pmc">PMC</a>, see 2.4. below) believes the individual is no longer able to responsibly discharge the duty of the role.
+<p>Roles are never rescinded because of inactivity—unless that inactivity is causing a problem for the project. Fortunately, our decision-making process means that inactivity is very rarely a problem. Roles will be rescinded if the <a href="#pmc">Project Management Committee</a> (or <a href="#pmc">PMC</a>, see 2.4. below) believes the individual is no longer able to discharge the duty of the role responsibly.
 
-<p>We understand that you have many roles in life. We use a hat metaphor to talk about these roles. For instance, you might have your work hat as well as several ASF hats. We expect you to know when to wear the appropriate hat, and to comport yourself in a manner befitting the interests of the Foundation when interacting with the project. Failure to do this is a serious dereliction of duty.
+<p>We understand that you have many roles in life. We use a hat metaphor to talk about these roles. For instance, you might have your work hat as well as several ASF hats. We expect you to know when to wear the appropriate hat, and to comport yourself in a manner befitting the interests of the foundation when interacting with the project. Failure to do this is considered a severe dereliction of duty.
 
-<p>Sometimes it is a good idea to tell people which hat you are wearing. For instance, the PMC Chair might start an informal email by stating they are not wearing the PMC Chair's hat, just to be clear about how the statements ought to be interpreted.
+<p>Sometimes it is a good idea to tell people which hat you are wearing. For instance, the PMC chair might start an informal email by explicitly stating that they are not wearing the PMC chair's hat. Doing so can help to clear up the meaning (and implication) of what is said.
```

Comment:
-0.5 on chair vs. Chair since it's an official role/title and recognized by the Foundation (though technically it's called the V.P., CouchDB at the Foundation level)